### PR TITLE
RHPAM-4421: VS Code Bundle Extension for IBAMOE

### PR DESCRIPTION
**Pull request by tiagobento**

### CODE REVIEW

File: packages/vscode-extension-kie-ba-bundle/package.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{
```
Comment (ederign) [inline packages/vscode-extension-kie-ba-bundle/package.json:1]: @tiagobento just to double check the logo

File: packages/vscode-extension-kie-ba-bundle/src/extension/extension.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2022 Red Hat, Inc. and/or its affiliates.
```
Comment (ederign) [inline packages/vscode-extension-kie-ba-bundle/src/extension/extension.ts:2]: just to double-check this.

**Review by ederign — COMMENTED** [review]
Some minor double checks.

File: packages/vscode-extension-kie-ba-bundle/package.json (hunk 1 above)
Comment (tiagobento) [reply packages/vscode-extension-kie-ba-bundle/package.json:1]: We're going to use the KIE logo for this one.

**Review by ederign — APPROVED** [review]

**Review by porcelli — COMMENTED** [review]
looks good to me!

**Pull request merged**